.github/actions/paths-filter/action.yml

```diff
@@ -155,6 +155,13 @@ outputs:
         github.event_name == 'schedule' ||
         steps.filter-common.outputs.openapi-change == 'true'
       }}
+  stable-branch-changes:
```

Comment:
💭 I have second thoughts about the naming here since (for me) the name implies to match all `stable/*` branches, but actually the expression only looks for "proper" aka Zeebe-related stable branches like `stable/13.23`... I'm not against it mechanically but would love a better name for that, or in lieu of that the description mentioning that fact like:

> Output whether the change is occurring on a stable/X.Y branch or not